src/main/java/mate/academy/springboot/swagger/repository/ProductRepository.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.springboot.swagger.repository;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.swagger.model.Product;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ProductRepository extends JpaRepository<Product, Long> {
+    List<Product> findAllByPriceBetween(BigDecimal fromPrice, BigDecimal toPrice,
+                                        PageRequest pageRequest);
```

Comment:
i think it's better to use pageable and turn PageRequestService into SortingUtil class or SortingService